docs/hub/sample-factory.md

```diff
@@ -0,0 +1,95 @@
+
+### Download Model Repository Directly
+
+Hugging Face repositories can be downloaded directly using `git clone`:
+
+```
+git clone <URL of HuggingFace Repo>
+```
+
+## Using Downloaded Models with Sample-Factory
+
+After downloading the model, you can run the models in the repo with the enjoy script corresponding to your environment. For example, if you are downloading a `mujoco-ant` model, it can be run with:
+
+```
+python -m sf_examples.mujoco_examples.enjoy_mujoco --algo=APPO --env=mujoco_ant --experiment=<repo_name> --train_dir=./train_dir
+```
+
+Note, you may have to specify the `--train_dir` if your local train_dir has a different path than the one in the `cfg.json`
+
+## Sharing your models
+### Using push_to_hub
+
+If you want to upload without generating evaluation metrics or a replay video, you can use the `push_to_hub` script:
+
+```
+python -m sample_factory.huggingface.push_to_hub -r <hf_repo_name> -u <hf_username> -d <experiment_dir_path>
+```
+
+The command line arguments are:
+
+- `-r`: The repo's name to save on Hub. This is the same as `hf_repository` in the enjoy script
+- `-u`: Your Hugging Face username or organization where to save the models
+- `-d`: The full path to your experiment directory to upload
+
+
+### Using enjoy.py
+
+You can upload your models to the Hub using your environment's `enjoy` script with the `--push_to_hub` flag. Uploading using `enjoy` can also generate evaluation metrics and a replay video.
+
+The evaluation metrics are generated by running your model on the specified environment for a number of episodes and reporting the mean and std reward of those runs.
+
+Other relevant command line arguments are:
+
+- `--hf_repository`: The repository to push to. Must be of the form `<username>/<repo_name>`. The model will be saved to `https://huggingface.co/<username>/<repo_name>`
+- `--max_num_episodes`: Number of episodes to evaluate on before uploading. Used to generate evaluation metrics. It is recommended to use multiple episodes to generate an accurate mean and std.
+- `--max_num_frames`: Number of frames to evaluate on before uploading. An alternative to `max_num_episodes`
+- `--no_render`: A flag that disables rendering and showing the environment steps. It is recommended to set this flag to speed up the evaluation process.
+
+You can also save a video of the model during evaluation to upload to the hub with the `--save_video` flag
+
+- `--video_frames`: The number of frames to be rendered in the video. Defaults to -1 which renders an entire episode
+- `--video_name`: The name of the video to save as. If `None`, will save to `replay.mp4` in your experiment directory
+
+For example:
+
+```
+python -m sf_examples.mujoco_examples.enjoy_mujoco --algo=APPO --env=mujoco_ant --experiment=<repo_name> --train_dir=./train_dir --max_num_episodes=10 --push_to_hub --hf_username=<username> --hf_repository=<hf_repo_name> --save_video --no_render
```

Comment:
The `hf_username` is not correct anymore right?

Reply:
Updated thanks